client/src/router/index.ts

```diff
@@ -36,7 +36,7 @@ const router = createRouter({
     },
     {
       name: "BallotVerificationLanding",
-      path: "/:locale/:electionSlug/verification",
+      path: "/:locale/:electionSlug/verify",
```

Comment:
Consider adding a redirect on the old URL, in case this has already been deployed to production, and someone has stored their old links.

Reply:
Verification was a new route I added so no one is using it so far. I just decided to use the same one we were already using instead of having two different paths for the same flow.